lib/src/render/image/image_upload_widget.dart

```diff
@@ -84,8 +93,47 @@ class _ImageUploadMenuState extends State<ImageUploadMenu> {
     super.dispose();
   }
 
+  void _pickFiles() async {
+    _resetState();
+    try {
+      _paths = (await FilePicker.platform.pickFiles(
+              type: FileType.image,
+              onFileLoading: (FilePickerStatus status) =>
+                  _logException(status.toString()),
+              allowedExtensions: ['jpg', 'png', 'gif'],
+              allowMultiple: true))
```

Comment:
```suggestion
              allowMultiple: true,))
```
Comma for formatting, will make it easier to read.
Also maybe a good idea to split it up instead of using `()?.files` as it makes it a bit harder to read. _(just my opinion)_